src/monmove.cpp

```diff
@@ -1070,6 +1070,8 @@ tripoint monster::scent_move()
         return { -1, -1, INT_MIN };
     }
 
+    const std::set<scenttype_id> tracked_scents = type->scents_tracked;
```

Comment:
```suggestion
    const std::set<scenttype_id> &tracked_scents = type->scents_tracked;
```
All the pointless copies.